centos.org/pipelines/foreman-discovery-image-build.groovy

```diff
@@ -43,8 +43,8 @@ pipeline {
 
     post {
         success {
-            archiveArtifacts artifacts: 'result/*tar', allowEmptyArchive: true
-            archiveArtifacts artifacts: 'result/*iso', allowEmptyArchive: true
+            archiveArtifacts artifacts: 'result/fdi*tar', allowEmptyArchive: true
+            archiveArtifacts artifacts: 'result/fdi*iso', allowEmptyArchive: true
```

Comment:
This is needed because lorax now requires an installation ISO to be present in the working directory which is named `install.iso`. We do not want to install netboot CentOS 8 Stream ISO to the destination folder.